### PR TITLE
Fixing repo reference in readme

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ This repository contains the PHP classes generated from protos contained in
 PHP Versions
 ----------------
 
-gax-php currently requires PHP 5.5 or higher.
+proto-client-php currently requires PHP 5.5 or higher.
 
 
 Contributing
```
Comment (michaelbausor) [inline README.md]: Change URL in contributing

Comment (garrettjonesgoogle) [inline README.md]: Fixed, and there was one more reference in License also.

**Comment by garrettjonesgoogle** [comment]
PTAL

**Comment by michaelbausor** [comment]
LGTM

**Pull request merged**